lib/os.js

```diff
@@ -86,7 +86,7 @@ function loadavg() {
 }
 
 function cpus() {
-  const data = getCPUs();
+  const data = getCPUs() || [];
```

Comment:
Can you please add a comment explaining why there is a fallback?

Reply:
Done